2019-03-Autumn/1_linked_lists/README.md

```diff
@@ -70,21 +107,77 @@ TODO :bug:
 
 TODO :bug:
 
-### 2. PROBLEM 2 TODO :bug:
-
-Source: TODO :bug:
-
-#### Naive/Simple Solution
-
-TODO :bug:
-
-#### Optimal Solution
-
-TODO :bug:
-
-#### Testing The Solutions OR Driver For Solution
-
-TODO :bug:
+### 2. Loop Detection
+
+Source: CTCI 2.8
+
+#### Solution for Detect Loop
+
+By using the *slow pointer and fast pointer* approach, we can guarantee that in 
```

Comment:
An aside: this is also called _tortoise and hare_ (not a blocking comment, just a fun factoid)